pomegranate/distributions/BernoulliDistribution.pyx

```diff
@@ -75,18 +75,40 @@ cdef class BernoulliDistribution(Distribution):
 			self.summaries[0] += w_sum
 			self.summaries[1] += x_sum
 
-	def from_summaries(self, inertia=0.0):
+	def from_summaries(self, inertia=0.0, pseudocount: float = 0.0):
 		"""Update the parameters of the distribution from the summaries."""
 
 		if self.summaries[0] < 1e-8 or self.frozen:
 			return
 
-		p = self.summaries[1] / self.summaries[0]
+		n: int = 2
+		p = (self.summaries[1] + pseudocount) / (self.summaries[0] + n * pseudocount)
 		self.p = self.p * inertia + p * (1-inertia)
 		self.logp[0] = _log(1-p)
 		self.logp[1] = _log(p)
 		self.summaries = [0.0, 0.0]
 
+	def fit(
```

Comment:
Not sure a new fit function is needed here. The other classes call the `from_summaries` method not the `fit` method regardless.

Reply:
The `Distribution.from_samples` calls fit, so it is helpful for doing
```Python
a = [0.0, 0.0, 0.0]
d = BernoulliDistribution.from_samples(a, pseudocount=1)
```
like in the unit test.